scripts/camViewer

```diff
@@ -248,16 +264,26 @@ if [ $# -ge 1 ]; then
     exit 1
 fi
 
-
-if [ "$hutch" == ""  ]; then
-    hutch=$(get_info --gethutch)
-    if [ "$hutch" == unknown_hutch ]; then
-        echo 'unknown hutch, specify using -H option'
-        usage
-        exit 1
+VH="/reg/g/pcds/epics-dev/spencera/engineering_tools/scripts/verify-hutch"
+
+hutch_check() {
+    hutch=$1
+    if [[ "$hutch" == "" ]]; then
+        hutch=$(get_info --gethutch)
+        if [[ "$hutch" == unknown_hutch ]]; then
+            if [[ $NUDGE -gt 0 ]]; then
+                echo 'To start/stop acquisition, you must be on the hutch specific machine'
+                exit 1 
+            else
+                echo 'unknown hutch, specify using -H option'
+                usage
+                exit 1
+            fi
+        fi
```

Comment:
A couple notes on the hutch check:

1. I don't think it needs to be a function if only run once immediately.
2. If there was a hutch passed, it should be checked using the verify-hutch script.
3. I recommend taking the nudge check out of here and maybe moved down to right above the acquire/stop/nudge section, and check the current hutch against the required hutch if possible. This might be complicated for shared cameras though, so just removing it might be easier.